scripts/filter.mjs

```diff
@@ -0,0 +1,24 @@
+import createFeed from "./createFeed.mjs";
+export default function filter(){
+  const filter = document.querySelector("select");
```

Comment:
```suggestion
  const sort = document.querySelector("select");
```

Reply:
Fixed the search function to do real time search and content updating instead of waiting for a submit to improve the ux, like you said in discord.